frontend/__tests__/pages/doctor/patient-list-overview.test.js

```diff
@@ -62,6 +62,7 @@ describe("onChange functions", () => {
     const component = shallow(<PatientListOverview patientList={patientList} />);
     it("onChange -> filterPatient + onSearch", () => {
         const mockEvent = {
+            // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
can you remove this eslint atrocity

Reply:
It gives the red squiggly line if we remove it ;-;